content/en/blog/_posts/2023-01-17-confidential-kubernetes.md

```diff
@@ -0,0 +1,312 @@
+---
+layout: blog
+title: "Confidential Kubernetes: Use Confidential Virtual Machines and Enclaves to improve your cluster security."
+date: 2023-01-17
+slug: "confidential-kubernetes"
+---
+
+**Authors:** Fabian Kammel (Edgeless Systems), (more to add...)
+
+In this blog post, we will introduce the concept of Confidential Computing (CC) to improve
+the security and privacy properties of any computing environment. Further, we will show how
+the Cloud-Native ecosystem, and Kubernetes in particular, can benefit from the new compute paradigm.
+
+Confidential Computing is not a new concept in the cloud-native world. The
+[Confidential Computing Consortium](https://confidentialcomputing.io/) is a Linux Foundation
+that already worked on
+[Defining and Enabling Confidential Computing](https://confidentialcomputing.io/wp-content/uploads/sites/85/2019/12/CCC_Overview.pdf).
+In the [Whitepaper](https://confidentialcomputing.io/wp-content/uploads/sites/85/2022/11/CCC-A-Technical-Analysis-of-Confidential-Computing-v1.2_updated_2022-11-02.pdf)
```

Comment:
Whitepaper link seems to be dead.. link to v1.3 works:
https://confidentialcomputing.io/wp-content/uploads/sites/85/2023/01/CCC-A-Technical-Analysis-of-Confidential-Computing-v1.3_Updated_November_2022.pdf